core/pom.xml

```diff
@@ -22,6 +22,7 @@
         <netbeans.hint.license>apache20</netbeans.hint.license>        
         <maven.build.timestamp.format>yyyy-MM-dd'T'HH:mm:ssZ</maven.build.timestamp.format>        
         <builddate>${maven.build.timestamp}</builddate>        
+        <geotools.version>10.8</geotools.version>
```

Comment:
I had used, mvn clean test verify
that seemed to use java 1.6 hence had to use previous version. 
Here tests are run against 1.7 and 1.8 , so will update to newer version.